### DISCUSSION TITLE
Exolve from .puz

### DISCUSSION THREAD

**Discussion by Joel-Sathiaraj** [viresh-ratnakar/exolve#68] [Q&A] [answered]
Could I have a sample file for .puz similar to how one exists for .ipuz ?
I am not able to import my .puz files. Console shows no error either.

**Comment by viresh-ratnakar** [comment, marked as answer]
I have added a .puz example file here:
https://github.com/viresh-ratnakar/exolve/blob/master/test-3x3.puz

There is an "exolve-player.html" file that can be used to open a .puz file
(or a .ipuz or exolve file). Did you try with that? You can access it at
this URL: https://viresh-ratnakar.github.io/exolve-player.html. Download
the test-3x3.puz file and open it with exolve-player (or drag-and-drop on
to it).

If it's possible, please email me (viresh at gmail dot com) your .puz file
and I can try to figure out what's going wrong.

Viresh
